middlewares/campaigns.js

```diff
@@ -33,14 +35,32 @@ exports.registerUser = async (req, res) => {
             transaction: t
         });
 
+        // TODO: add duplicate name checking; translit gsuite_email in case of umlauts
+        const gsuite_email = req.body.first_name+'.'+req.body.last_name+'@aegee.eu'
```

Comment:
spaces before and after plus, also semicolon at the end

Reply:
also domain should be moved to constants I'd say

Reply:
yep I hurried a push because I didn't want to block development